ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -310,9 +312,19 @@ + (ADAuthenticationResult *)processBrokerResponse:(NSURL *)response
         MSIDOauth2Factory *factory = [MSIDAADV1Oauth2Factory new];
         MSIDDefaultTokenCacheAccessor *otherAccessor = [[MSIDDefaultTokenCacheAccessor alloc] initWithDataSource:dataSource otherCacheAccessors:nil factory:factory];
         MSIDLegacyTokenCacheAccessor *cache = [[MSIDLegacyTokenCacheAccessor alloc] initWithDataSource:dataSource otherCacheAccessors:@[otherAccessor] factory:factory];
+        
+        NSString *userId = [[[result tokenCacheItem] userInformation] userId];
```

Comment:
Do we want to use the dot notation rather than brackets here to make it easier to understand :)

Reply:
+1 :)

Reply:
addressed :)